src/components/sidebar/UserInfo.jsx

```diff
@@ -13,8 +13,6 @@ export const UserInfo = props => {
           {email}
         </Text>
       </Box>
-      <Spacer />
-      <UpDownIcon />
```

Comment:
:0

Reply:
Lol I'm trying to fix it still so it's in the hover it ain't deleted tho